math/minuit/doc/index.md

```diff
@@ -1,6 +1,7 @@
 \defgroup MinuitOld TMinuit
 \ingroup Math
 \brief The Minuit Minimization package.
+\deprecated Consider switching to Minuit2
```

Comment:
```suggestion
```
I would not do this in this PR. The original Minuit is still used all over the place. Deprecating means that it will be removed in the long run, which is just something we can't do for the original Minuit.

Reply:
Maybe I can convert it to a note?